tf2_ros/test/listener_unittest.cpp

```diff
@@ -49,22 +50,17 @@ void generate_rand_vectors(double scale, uint64_t runs, std::vector<double>& xva
   }
 }
 
-
-using namespace tf2;
-
-TEST(tf2_ros_transform, transform_listener)
+TEST(tf2_ros_test_listener, transform_listener)
 {
-  tf2_ros::Buffer buffer;
-  tf2_ros::TransformListener tfl(buffer);
-  
+  rclcpp::Clock::SharedPtr clock = std::make_shared<rclcpp::Clock>(RCL_SYSTEM_TIME);
 
+  tf2_ros::Buffer buffer(clock);
+  tf2_ros::TransformListener tfl(buffer);
```

Comment:
There are no `EXPECT` or `ASSERT` statements in this test, so it basically can never fail unless it throws an exception or crashes.  What are we trying to test here?

Reply:
Not really sure about this. I checked in https://github.com/ros/geometry2/blob/melodic-devel/tf2_ros/test/listener_unittest.cpp and there are no `EXPECT` or `ASSERT` too

Reply:
Sorry for the delay here.  Yeah, I see that there are non in the old one, but I think it would probably be a good idea to add something to EXPECT here.  Otherwise we aren't testing all that much.

Reply:
Added some EXPECTs